aws-lc-rs/src/ptr.rs

```diff
@@ -31,14 +34,14 @@ impl<P: Pointer> LcPtr<P> {
     }
 }
 
-impl<P: Pointer> Drop for LcPtr<P> {
+impl<P: Pointer> Drop for ManagedPointer<P> {
     #[inline]
     fn drop(&mut self) {
         self.pointer.free();
     }
 }
 
-impl<P: Pointer + Copy> LcPtr<P> {
+impl<P: Pointer + Copy> ManagedPointer<P> {
```

Comment:
Why do we need `P` to be `Copy`? This might be an artifact of some method we wrote that has since been removed.(?)